helpers.py

```diff
@@ -1,8 +1,10 @@
-from webtodotxt import app
+import auth_backends
 import storage_backends
+from webtodotxt import app
```

Comment:
Can you put this line at the very beginning of this file?

Reply:
Addressed in latest commits.